Library/Homebrew/formulary.rb

```diff
@@ -411,6 +413,20 @@ def self.factory(
     formula
   end
 
+  # Map a formula name to a bottle archive. This mapping will be used by {Formulary::factory}
+  # to allow formulae to be loaded automatically from their bottle archive without
+  # needing to exist in a tap or be passed as a complete filepath. For example,
```

Comment:
Could it be used now if an environment variable is set?

Reply:
Used how? What would the environment variable do? Right now there's no place where it makes sense to set up the mapping outside of Homebrew/json. So I don't think adding an environment variable would do much here. As soon as this and https://github.com/Homebrew/homebrew-json/pull/7 are merged, anyone using Homebrew/json would use this